circleci/docker.go

```diff
@@ -97,6 +97,15 @@ func findNewestDockerVersion(currentVersion string, parameters *map[string]*yaml
 			continue
 		}
 
+		if len(v.Prerelease()) > 0 {
+			log.Debug().Msgf("version %s skipped, prerelease", v.Original())
+			continue
+		}
+		if trimmed := TrimSemver(currentv.Original(), v.Original()); len(trimmed) != len(v.Original()) {
```

Comment:
Won't this part `len(trimmed) != len(v.Original()` not work if we go from version `1.99.0` to version `1.100.0`

Reply:
It works but It's a bit unintuitive. 
It trims the the new version to the same amount of segments as the currentv. Afterwards i'm checking if the trimmed and original are identical.
So in your example, if current was `1.99` it will trim `1.100` to `1.100` and then compare those two.

Reply:
If we have the following:
```go
trimmed := "1.99"
v.Original() := "1.991"
```

in this example `v.Original()` is newer than current version(`trimmed`) but according to this if statement it would be skipped, right? or am I completely wrong?

Reply:
TrimSemver trims the 2nd parameter to the same patch level as the fist param:
fx:
```go
TrimSemver("1.99", "1.999.1") -> "1.999"
TrimSemver("1.99.1", "1.999.1") -> "1.999.1"
```